@uportal/content-carousel/src/App.ts

```diff
@@ -20,6 +20,7 @@ export default class App extends Vue {
     infinite: true,
     arrows: true,
     dots: true,
+    variableWidth: true,
```

Comment:
neat, :+1: 
could this be added to the example in `@uportal/content-carousel/README.md` and/or `docs/en/components/content-carousel/demo.html`?